xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayText.h

```diff
@@ -8,150 +8,23 @@
 
 #pragma once
 
+#include "../../DVDSubtitles/DVDSubtitlesLibass.h"
```

Comment:
Relative include intended?

Reply:
not really i have copied from existing code, not knowing the reason of this relative case i have kept as is
i will change